packages/flame/lib/src/widgets/sprite_widget.dart

```diff
@@ -78,15 +84,19 @@ class _SpriteWidget extends StatelessWidget {
   /// The positioning [Anchor] for the [sprite]
   final Anchor anchor;
 
+  /// The angle to rotate this [sprite], in rad. (default = 0)
+  final double angle;
+
   const _SpriteWidget({
     required this.sprite,
     this.anchor = Anchor.topLeft,
+    this.angle = 0,
   });
 
   @override
   Widget build(_) {
     return Container(
```

Comment:
Never dawned to me before, but why are we wrapping this in a container?

Reply:
no idea...

Reply:
@erickzanardo will know